components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -754,14 +754,14 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
                                   }`}
                                 />
                               )}
-                              <p
-                                className="text-light-slate-11 truncate max-w-[14rem] xs:max-w-[16rem] text-xs xs:text-sm cursor-pointer hover:text-orange-600 transition"
+                              <button
+                                className="outline-none bg-none text-light-slate-11 truncate max-w-[14rem] xs:max-w-[16rem] text-xs xs:text-sm cursor-pointer hover:text-orange-600 transition"
                                 onClick={() => {
                                   window.open(suggestion.url, "_blank");
                                 }}
                               >
                                 {suggestion.title}
-                              </p>
+                              </button>
```

Comment:
This does fix the warning, but this is a link so it should be an anchor tag instead of a button with an onClick opening the link. Leverage the `target` prop and retain the outline.

```suggestion
                              <a
                                href={suggestion.url}
                                target="_blank"
                                className="truncate max-w-[14rem] xs:max-w-[16rem] text-xs xs:text-sm"
                              >
                                {suggestion.title}
                              </a>
```